src/Simulation/Core/Submitters/IQuantumMachine.cs

```diff
@@ -1,34 +1,24 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-using System.Threading.Tasks;
+using Microsoft.Quantum.Runtime.Submitters;
 using Microsoft.Quantum.Simulation.Core;
+using System;
+using System.Threading.Tasks;
 
 namespace Microsoft.Quantum.Runtime
 {
     /// <summary>
-    /// Interface that a quantum machine must implement.
+    /// An interface for submitting Q# programs to Azure.
     /// </summary>
-    public interface IQuantumMachine
+    [Obsolete("Replaced by IQSharpSubmitter.")]
+    public interface IQuantumMachine : IAzureSubmitter
     {
-
         /// <summary>
         /// Function that configures a job object before submission.
         /// </summary>
         public delegate void ConfigureJob(object job);
```

Comment:
How come ConfigureJob is no longer needed? Or did it move somewhere and I missed it?

Reply:
This was introduced originally to be able to set an arbitrary state to the job object before submission. This was mostly used for testing and development since it required knowledge of the internal implementation of the submission and the Job object.

Since this is a public interface, I think it makes sense not to keep providing this option.